yarn-project/bb-prover/src/bb/execute.ts

```diff
@@ -95,11 +96,23 @@ export function executeBB(
 
     bb.stdout.on('data', data => {
       const message = data.toString('utf-8').replace(/\n$/, '');
-      logger(message);
+      pidusage(bb.pid!, (err, stats) => {
```

Comment:
This allows us to gather RAM info when running the native prover, maintaining the same benchmarking system